config/variables.tf

```
@@ -26,4 +20,10 @@ variable "tf_url" {
   type        = string
   description = "The url of the keycloak to perform the terraform actions on"
   sensitive   = true
+}
+
+variable "cal_url" {
```

Can’t spot where this is used, but if we still need it, please can we rename it for clarity?

```suggestion
variable "callisto_url" {
```

---

Ah it was in the CSP right. Dang.

---

I've renamed the variable to something that's more immediately obvious